js/piwik.js

```diff
@@ -4962,14 +5001,23 @@ if (typeof Piwik !== 'object') {
                 /**
                  * Set heartbeat (in seconds)
                  *
-                 * @param int minimumVisitLength
-                 * @param int heartBeatDelay
+                 * @param int heartBeatDelay Defaults to 15.
                  */
-                setHeartBeatTimer: function (minimumVisitLength, heartBeatDelay) {
-                    var now = new Date();
+                setHeartBeatTimer: function (heartBeatDelay) {
```

Comment:
Probably not everyone likes it, I'd personally name it `heartBeatDelayInSeconds` and also mention it in doc block that it is seconds since one is used to deal with `ms` eg in `setTimeout` etc.